homeassistant/components/flexpool/__init__.py

```diff
@@ -0,0 +1,41 @@
+"""The Flexpool integration."""
+import asyncio
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+PLATFORMS = ["sensor"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Flexpool component."""
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up Flexpool from a config entry."""
+
+    for platform in PLATFORMS:
+        hass.async_create_task(
+            hass.config_entries.async_forward_entry_setup(entry, platform)
+        )
+
+    return True
+
+
+async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Unload a config entry."""
+    unload_ok = all(
+        await asyncio.gather(
+            *[
+                hass.config_entries.async_forward_entry_unload(entry, platform)
+                for platform in PLATFORMS
+            ]
+        )
+    )
```

Comment:
```suggestion
    unload_ok = await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
```

Reply:
For some reason this introduced the following issue:

```
    hass.data[DOMAIN].pop(entry.entry_id)
KeyError: 'flexpool'
```
Not sure if async_unload_platforms does something different which doesn't require me to pop anymore?